test/framework/robot.py

```diff
@@ -809,6 +810,46 @@ def test_find_resolved_modules(self):
 
         self.assertTrue(new_avail_modules, ['nodeps/1.2.3', 'onedep/3.14-goolf-1.4.10'])
 
+    def test_tweak_robotpath(self):
+        """Test that the robot correctly resolves the dependencies of tweaked easyconfigs. Tweaked
+        easyconfigs take priority, but tweaked dependencies are only used on an as-needed basis"""
+
+        test_easyconfigs = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'easyconfigs', 'test_ecs')
+
+        # Create directories to store the tweaked easyconfigs
+        eb_tmpdir = tempfile.gettempdir()
+        tweaked_ecs_paths, pr_path = alt_easyconfig_paths(eb_tmpdir, tweaked_ecs=True)
```

Comment:
@ocaisa please use `self.test_prefix` instead of `eb_tmpdir`, which is a sandbox created for every test that is also cleaned up automagically